package.json

```diff
@@ -36,10 +36,14 @@
     "gulp-clean": "~0.3.1",
     "gulp-uglify": "~1.1.0",
     "gulp-shell": "~0.3.0",
-    "es6-promise": "~2.0.1"
+    "es6-promise": "~2.0.1",
+    "uglify-js": "~2.4.17"
   },
   "scripts": {
+    "all": "gulp all",
+    "build": "gulp build",
     "test": "gulp test",
+    "test:watch": "gulp test:watch",
```

Comment:
I'm not aware of that if that's the case. It's not mentioned in the [docs](https://docs.npmjs.com/misc/scripts). 

As far as I know it just executes the string (as shell in Unix and as cmd(?) in Windows)